src/drivers/imx/sai.c

```diff
@@ -229,6 +232,7 @@ static inline int sai_set_config(struct dai *dai, struct ipc_config_dai *common_
 	uint32_t val_cr2 = 0, val_cr4 = 0, val_cr5 = 0;
 	uint32_t mask_cr2 = 0, mask_cr4 = 0, mask_cr5 = 0;
 	uint32_t clk_div;
+	int tdm_mode;
```

Comment:
how about directly use bool tdm_mode? false naturally means disable, and true is enable.

Reply:
@btian1 yes, this makes sense.